drift/pubspec.yaml

```diff
@@ -16,33 +16,33 @@ environment:
   sdk: ">=3.4.0 <4.0.0"
 
 dependencies:
-  async: ^2.5.0
-  convert: ^3.0.0
-  collection: ^1.15.0
-  meta: ^1.3.0
+  async: ^2.11.0
+  convert: ^3.1.0
+  collection: ^1.18.0
+  meta: ^1.12.0
```

Comment:
I don't really see the point of these updates? Are we inadvertently using features added in these later versions? This just restricts dependencies for no reason (or at least no reason apparent to me)?

I suppose technically we're incompatible with the oldest version declared here due to other dependencies which might require newer versions, but that doesn't mean that we'd have to update them.

Reply:
They all correspond to pinned dependencies in Flutter version 3.22(Dart 3.4)

Reply:
If it is critical, I will lower these versions back.